README.md

```diff
@@ -73,43 +72,37 @@ In particular:
 - Thibaut Mattio (Tarides)
 - Anil Madhavapeddy (University of Cambridge)
 
-For the groundwork on rethinking the sitemap, user flows, new content, design, and frontend and package docs!
-
+For the work on the package site infrastructure and UI:
 - Jon Ludlam (OCaml Labs)
 - Jules Aguillon (Tarides)
 - Lucas Pluvinage (Tarides)
 
-For the work on the package site infrastructure and UI!
-
+For meticulously going through the website to find issues:
 - Paul-Elliot Anglès d’Auriac (Tarides)
 
-For meticulously going through the website to find issues.
-
+For the work on the frontend designs and bringing them to life:
 - Isabella Leandersson (OCaml Labs)
 - Asaad Mahmood (Tarides)
 
-For the work on the designs and bringing them to life on the frontend!
-
+For the work on the new content and reviewing the existing one:
 - Christine Rose (Tarides)
 - Isabella Leandersson (OCaml Labs)
 
-For the work on the new content and reviewing the existing one!
-
-We’d also like to thank the major funders who supported work on revamping the website: grants from the Tezos Foundation and Jane Street facilitated the bulk of the work. Thank you, and if anyone else wishes to help support it on an ongoing basis then donations to the OCaml Software Foundation and grants to the maintenance teams mentioned above are always welcomed.
+We’d also like to thank the major funders who supported work on revamping the website. Grants from the Tezos Foundation and Jane Street facilitated the bulk of the work. Thank you! And if anyone else wishes to help support it on an ongoing basis, then donations to the OCaml Software Foundation and grants to the maintenance teams mentioned above are always welcomed.
 
 ## Contributing
 
-We'd love your help improving ocaml.org!
+We'd love your help improving OCaml.org!
 
 See our contributing guide in [`CONTRIBUTING.md`](./CONTRIBUTING.md)
```

Comment:
This is the second time in the README this is mentioned and linked. Do you want it there twice?

Reply:
You are right @christinerose, the first occurrence should be removed.

Reply:
Fixed